docs/agents/sematext-agent/processes/sensitive-data.md

```diff
@@ -0,0 +1,7 @@
+The agent ensures any sensitive data such as credentials or tokens are scrubbed before shipping the data to Sematext Cloud.
+
+For example, launching a Python process with `--password` and `MYSQL_PASSWORD` arguments passed to it, would result in the command line metadata attribute being masked as following:
```

Comment:
Do/should we list all patterns we look for?  If we don't cover something that one uses, can a person add his/her pattern somewhere?

Reply:
We can list the patterns, but currently we don't support custom patterns :(

Reply:
+1 for listing patterns and customer patterns is something we can consider adding IF we see demand for this.